Source/Core/InputCommon/ControllerEmu/ControlGroup/ControlGroup.h

```diff
@@ -89,9 +89,9 @@ class ControlGroup
     return std::copysign(std::max(T{0}, std::abs(input) - deadzone) / (T{1} - deadzone), input);
   }
 
-  const std::string name;
-  const std::string ui_name;
-  const GroupType type;
+  const std::string m_name;
+  const std::string m_ui_name;
+  const GroupType m_type;
```

Comment:
This seems inconsistent. If you're going to change the member names, it's better to update all of them.

Reply:
@BhaaLseN mentioned #8684 that he prefers to only change the fields impacted.

Reply:
Using the m_ prefix feels... wrong for (public) *struct* member variables. I'd just add a _ suffix to whatever constructor arguments require them.